.github/workflows/automationTemplate.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
           git add -A
           git commit -m "Add new SDK versions" | true
           gitCommitExitCode=${PIPESTATUS[0]}
-          if [[ $gitCommitExitCode = 0 ]]; then
+          if [[ $gitCommitExitCode != 0 ]]; then
```

Comment:
Should this condition be inverted? I expected the gitCommitExitCode to be >= 1 if there was nothing to commit, in which case we would want to cancel the workflow.

Reply:
Looking at the rest of the PR, I'm pretty sure this was done for testing purposed. :)

Reply:
AH yes! Excellent catch :)

Reply:
Updated

Reply:
Thanks